packages/dds/tree/src/test/domains/json/jsonGeneratorUtils.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * This file contains a series of utility functions intended to assist with generating random data.
+ */
+
+import { makeRandom } from "@fluid-internal/stochastic-test-utils";
+
+const englishAlphabet = "abcdefghijkmnopqrstuvwxyzABCDEFGHJKLMNPQRSTUVWXYZ";
+
+export function getRandomStringByCharCode(random = makeRandom(), minLen: number, maxLen: number,
+ charCodeMin: number, charCodeMax: number) {
+    const stringLength = (minLen < maxLen) ? random.integer(minLen, maxLen) : minLen;
+    let string = "";
+    for (let i = 0; i < stringLength; i++) {
+        string += String.fromCharCode(charCodeMin + random.real() * (charCodeMax - charCodeMin + 1));
+    }
+    return string;
+}
+
+// Returns either an alphanumeric string or an alpha string within the specified length range
+export function getRandomEnglishString(random = makeRandom(), includeNumbers: boolean, minLen: number, maxLen: number) {
+    const stringLength = (minLen < maxLen) ? random.integer(minLen, maxLen) : minLen;
+    if (includeNumbers) {
+        return random.string(stringLength);
+    } else {
+        return random.string(stringLength, englishAlphabet);
+    }
+}
+
+export function getRandomNumberString(random = makeRandom(), minLen: number, maxLen: number) {
+    return getRandomStringByCharCode(random, minLen, maxLen, 0x0030, 0x0039);
+}
```

Comment:
random has a '.string()' helper that will construct string given an "alphabet", where an alphabet is a string containing the valid characters:

// A string of Arabic numerals of length [1..10]:
const numberString = random.string(
    /* length: */ random.integer(1, 10),
    /* alphabet: */ "0123456789");

You could modify 'getRandomStringByCharCode' to construct alphabets for use with 'random.string':

```ts
// Constructs a string containing all character is the range [charMin..charMax] (inclusive).
export function getCharRange(charMin: string, charMax: string) {
    const min = charMin.codePointAt(0);
    const max = charMax.codePointAt(0);

    let string = "";
    for (let i = min; i <= max; i++) {
        string += String.fromCodePoint(min + i);
    }

    return string;
}
```

Reply:
Good idea. I switched to this as of the 3rd commit (hash 133796)